Packs/CarbonBlackDefense/Integrations/CarbonBlackDefenseV2/CarbonBlackDefenseV2_description.md

```diff
@@ -0,0 +1,33 @@
+## Carbon Black Cloud Endpoint Standard
+
+*Carbon Black Cloud Endpoint Standard is the new name for the product formerly called CB Defense.*
+
+### Create Access Levels and API Keys
+
+There are two versions of the API - V3 and V6. V6 includes all the APIs except for the Policy APIs. Therefore, in addition to creating an API key and API secret key, you also need to create a custom API key and a custom API secret key.
```

Comment:
the custom is only for the policy API ? Id so, please mention it, something like:
```suggestion
There are two versions of the API - V3 and V6. V6 includes all the APIs except for the Policy APIs. Therefore, in addition to creating an API key and API secret key, you also need to create a custom API key and a custom API secret key for the policy related commands.
```